locales-pending/premium.ftl

```diff
@@ -38,6 +38,14 @@ exposure-chart-legend-heading-nr = Number
 # Variables:
 #   $nr (number) - Number of a particular type of exposure found for the user
 exposure-chart-legend-value-nr = { $nr }x
+exposure-chart-caption = This chart shows how many times your info is actively exposed.
+
+modal-active-number-of-exposures-title = About your number of active exposures
+# Variables:
+#   $limit (number) - Number of email addresses included in the plan
+modal-active-number-of-exposures-part-one = This chart includes the total number of times we found each type of data exposed across all data broker profiles and all data breaches for up to { $limit } email addresses that you are currently monitoring.
```

Comment:
Please make this a plural string (see issue [here](https://github.com/mozilla/blurts-server/issues/3240#issuecomment-1645737650))